apis/aws.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	cachev1alpha2 "github.com/crossplaneio/stack-aws/apis/cache/v1beta1"
 	computev1alpha2 "github.com/crossplaneio/stack-aws/apis/compute/v1alpha2"
-	databasev1alpha2 "github.com/crossplaneio/stack-aws/apis/database/v1alpha2"
+	databasev1alpha2 "github.com/crossplaneio/stack-aws/apis/database/v1beta1"
```

Comment:
I am realizing my `DbSubnetGroup` PR is now complicating this work here :/ @negz @muvaf do you think we should have a `v1beta` and `v1alpha2` (which I will be updating shortly to `v1alpha3`) directory under `database` or also bring `DbSubnetGroup` along to `v1beta1`?

Reply:
> do you think we should have a v1beta and v1alpha2 (which I will be updating shortly to v1alpha3) directory under database or also bring DbSubnetGroup along to v1beta1?

Let's keep DbSubnetGroup at v1alpha2 - I think it's close but needs a few small touches to reach what we consider v1beta1 readiness.

Reply:
SGTM